src/Data/Biapplicative.hs

```diff
@@ -454,3 +464,24 @@ instance Biapplicative Const where
   {-# INLINE bipure #-}
   Const f <<*>> Const x = Const (f x)
   {-# INLINE (<<*>>) #-}
+
+-- | An analogue of @"Control.Applicative.Backwards".'Control.Applicative.Backwards.Backwards'@
+-- for bifunctors.
+newtype Backwards p a b = Backwards { forwards :: p a b }
+  deriving (Show, Read, Eq, Ord, Functor, Base.Bifunctor)
+
+instance Biapplicative p => Biapplicative (Backwards p) where
```

Comment:
It would be helpful to include a Haddock comment on this instance which states that the actions are performed in the opposite order, similar to the [comment on `Control.Applicative.Backwards.Backwards`' `Applicative` instance](https://hackage.haskell.org/package/transformers-0.6.0.2/docs/Control-Applicative-Backwards.html). Similarly for `Backwards`' `Applicative` instance as well.